src/main.rs

```diff
@@ -356,6 +365,10 @@ fn main() {
             args.raw = true;
             eprintln!("Warning: The -m / --markdown flag is deprecated, use -r / --raw instead");
         }
+        if args.os.is_some() && args.platform.is_none() {
```

Comment:
But shouldn't we print the warning all the time? Might be confusing if passing both doesn't trigger any alarms. How about something like:
```rust
if args.os.is_some() {
  // print the warning
}
args.platform = args.platform.or(args.os);
```

Reply:
Ah, you're right, that makes sense.